package.json

```diff
@@ -140,5 +140,8 @@
 		"clsx": "1.2.1",
 		"js-cookie": "3.0.1",
 		"rxjs": "7.5.4"
+	},
+	"resolutions": {
+		"**/@types/react": "17.0.39"
```

Comment:
I noticed that when running `tsc --noEmit` there were a lot of TS errors reported. I was caused by multiple instances of `@types/react` in node_modules. This line fixes the issue.